test/unit/DomainTrustProcessorTest.cs

```diff
@@ -53,52 +54,50 @@ public void DomainTrustProcessor_EnumerateDomainTrusts_HappyPath()
         }
 
         [Fact]
-        public void DomainTrustProcessor_EnumerateDomainTrusts_SadPaths()
+        public async Task DomainTrustProcessor_EnumerateDomainTrusts_SadPaths()
         {
-            var mockUtils = new Mock<MockLDAPUtils>();
+            var mockUtils = new Mock<MockLdapUtils>();
             var searchResults = new[]
             {
-                new MockSearchResultEntry("CN\u003dexternal.local,CN\u003dSystem,DC\u003dtestlab,DC\u003dlocal",
+                LdapResult<IDirectoryObject>.Ok(new MockDirectoryObject("CN\u003dexternal.local,CN\u003dSystem,DC\u003dtestlab,DC\u003dlocal",
                     new Dictionary<string, object>
                     {
                         {"trustdirection", "3"},
                         {"trusttype", "2"},
                         {"trustattributes", 0x24.ToString()},
                         {"cn", "external.local"},
                         {"securityIdentifier", Array.Empty<byte>()}
-                    }, "", Label.Domain),
-                new MockSearchResultEntry("CN\u003dexternal.local,CN\u003dSystem,DC\u003dtestlab,DC\u003dlocal",
+                    }, "","")),
```

Comment:
Is it possible to set default values of empty strings for these last two params in MockDirectoryObject?

Reply:
Absolutely